addon/models/new-platform-flexberry-g-i-s-layer-metadata.js

```diff
@@ -2,19 +2,12 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import BaseModel from 'ember-flexberry/models/base';
 import Proj from 'ember-flexberry-data';
+
 let Model = BaseModel.extend({
   name: DS.attr('string'),
   type: DS.attr('string'),
   coordinateReferenceSystem: DS.attr('string'),
   settings: DS.attr('string'),
-  createTime: DS.attr('string'),
-  creator: DS.attr('string'),
-  editTime: DS.attr('string'),
-  editor: DS.attr('string'),
-  validations: {
-    name: { presence: true },
-    type: { presence: true }
-  },
```

Comment:
No, validations where removed, because:
a) model itself is not a right place for validations (it should be in edit form controller, or in edit-dialog component);
b) I was unable to save even already existing models, because of validation errors;

I'll create seperate task for validation rules, and related messages/summary on maps/layers edit forms.